gpii/node_modules/settingsHandlers/src/WebSocketsSettingsHandler.js

```diff
@@ -0,0 +1,154 @@
+/*!
+  GPII Web Sockets Settings Handler
+
+  Copyright 2014 Emergya
+
+  Licensed under the New BSD license. You may not use this file except in
+  compliance with this License.
+
+  You may obtain a copy of the License at
+  https://github.com/gpii/universal/LICENSE.txt
+*/
+
+/*global require*/
+
+(function () {
+
+    "use strict";
+
+    var fluid = require("infusion"),
+        gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.settingsHandlers");
+    fluid.registerNamespace("gpii.settingsHandlers.WebSocketsSettingsHandler");
+
+    // This global object keeps in memory the connected clients.
+    // Clients are added and removed respectively through addBrowser and
+    // removeBrowser functions
+    //
+    gpii.settingsHandlers.WebSocketsSettingsHandler.clients = {};
```

Comment:
Hi, this looks OK, apart from the weird "app" namespace (should be under gpii) - I think the best way will be to have two separate instances of the component - one as a conventional subcomponent of the LifecycleManager, and one mounted for the settings handlers at a global path - there's no need for them to share a model or a client list, right? And I assume they will be running separate services on separate URLs.